README.yaml

```diff
@@ -71,7 +70,18 @@ usage: |-
   ```hcl
   module "example" {
```

Comment:
```suggestion
  module "aws_config" {
```

Reply:
add 
```
For a complete example, see [examples/complete](examples/complete).
  For automated tests of the complete example using [bats](https://github.com/bats-core/bats-core) and [Terratest](https://github.com/gruntwork-io/terratest)
  (which tests and deploys the example on Datadog), see [test](test).
```
Similar to https://github.com/cloudposse/terraform-aws-service-control-policies/blob/master/README.yaml#L101

Reply:
0a0ade1ac5feb7ed24ae631f6a840d382f5cc19c